### PR TITLE
Allow to override the external dependencies.

### PR DESCRIPTION
**Pull request by woodspire**
Allow to override the external dependencies.

This way, it's possible to use local libraries.

### CODE REVIEW

**Comment by freekmurze** [comment]
I like that added functionality.

Could you add entries for these (+ the default) in the config file?

**Comment by woodspire** [comment]
> I like that added functionality.
> 
> Could you add entries for these (+ the default) in the config file?

sure

File: src/DashboardServiceProvider.php [hunk 1]
```diff
@@ -15,8 +15,8 @@ class DashboardServiceProvider extends ServiceProvider
     public function boot()
     {
         $this->app->make(Dashboard::class)
-            ->script('https://cdn.jsdelivr.net/gh/alpinejs/alpine@v2.x.x/dist/alpine.min.js')
-            ->stylesheet('https://rsms.me/inter/inter.css')
+            ->script(config('dashboard.scripts.alpinejs', 'https://cdn.jsdelivr.net/gh/alpinejs/alpine@v2.x.x/dist/alpine.min.js'))
```
Comment (freekmurze) [inline src/DashboardServiceProvider.php:18]: The defaults can be removed here now.

Comment (woodspire) [reply src/DashboardServiceProvider.php]: done

**Comment by freekmurze** [comment]
Thanks!

**Pull request merged**